### DISCUSSION TITLE
ComfyUi embedded informations in pictures

### DISCUSSION THREAD

**Discussion by wernerDahlen** [comfyanonymous/ComfyUI#1976] [Q&A]
It gives the information from another image when I drag the image into the browser window. When I put the images into the png info in Automatic1111, the information is reflected correctly, but not in ComfyUi. What's the problem?

**Comment by snomiao** [comment]
You could try check and compare the exif info with this tool

https://comfyui-embeded-workflow-editor.vercel.app/
